tensorflow/lite/arena_planner.cc

```diff
@@ -297,7 +297,13 @@ TfLiteStatus ArenaPlanner::CalculateAllocationOfInternalTensors(
 
 TfLiteStatus ArenaPlanner::CalculateDeallocationOfInternalTensors(
     int node_index) {
-  if (node_index < graph_info_->num_nodes()) {
+  // For the case if the graph is empty the node index can be negative since we
+  // substract from the active node, so the node_index can be zero for those
+  // cases
+  if (node_index < 0) {
```

Comment:
Hmm, now I wonder if it would be better to catch this at the callsite in CalculateAllocations? Rather than silently succeeding?

Reply:
@jdduke , thanks for the comments, i have shift the check to CalculateAllocations, kindly check.

Regards
Amit